homeassistant/components/tplink/entity.py

```diff
@@ -50,8 +50,3 @@ def __init__(
             sw_version=device.hw_info["sw_ver"],
             hw_version=device.hw_info["hw_ver"],
         )
-
-    @property
-    def is_on(self) -> bool:
-        """Return true if switch is on."""
-        return bool(self.device.is_on)
```

Comment:
`is_on` isn't a base entity property. It comes from `ToggleEntity` so it shouldn't have been here

https://github.com/home-assistant/core/blob/6525dad57a575d131265226465fbd5c08433f9d4/homeassistant/helpers/entity.py#L1605